core/.js/src/main/scala/io/udash/bindings/modifiers/ValueModifier.scala

```diff
@@ -21,11 +24,19 @@ trait ValueModifier[T] extends Binding with DOMManipulator {
       killNestedBindings()
 
       val oldEls = elements
-      elements = {
-        if (checkNull && propertyValue == null) emptyStringNode()
-        else builder(propertyValue, nestedInterceptor)
+      val newEls: Seq[Node] =
+        builder(propertyValue, nestedInterceptor)
+          .optIf(!checkNull || propertyValue != null)
+          .filter(_.nonEmpty)
+          .getOrElse(emptyStringNode())
+
+      elements = if (newEls.size == 1 && newEls.head.isInstanceOf[DocumentFragment]) {
```

Comment:
I'm afraid this condition is not sufficient. Please consider your slightly modified test case:
```scala
val p = SeqProperty.blank[String]
val template = div(produce(p)(s => Seq(
  "test".render,
  s.render
))).render
template.outerHTML shouldBe "<div>test</div>"

p.set(Seq("A", "B", "C"))
template.outerHTML shouldBe "<div>testABC</div>"
```
which results in:
```
[info] - should handle SeqFrag update *** FAILED ***
[info]   scala.scalajs.js.JavaScriptException: NotFoundError: Node was not found
```